controllers/response.go

```diff
@@ -87,15 +88,19 @@ var GetResponses = func(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	formid := mux.Vars(r)["formid"]
+	id := mux.Vars(r)["formid"]
+	data := strings.Split(id, "-")
+	formid := data[0]
+	token := data[1]
 
 	// Check privileges
 	form := &models.Form{}
 	collection := u.Collection("forms")
 	objID, _ := primitive.ObjectIDFromHex(formid)
 	filt := bson.M{"$and": bson.A{
 		bson.M{"_id": objID},
-		bson.M{"creator": rno}}}
+		bson.M{"creator": rno},
```

Comment:
Don't need `creator` here anymore since we want anyone with the token to be able to view the responses.

Reply:
There is already a check for `rno` and returns if null in [`form.go:22`](https://github.com/pulsejet/go-cerium/blob/master/controllers/form.go#L22).
Do I have to update that to send back a 401 response?

Reply:
Yeah, but I'll do that separately anyway cause it isn't related to this PR